### PR TITLE
Add version history for 5.4.1

### PR DESCRIPTION
**Pull request by dgault**
For a list of included PR's:
Versioning clarifications - https://github.com/openmicroscopy/bioformats/pull/2813
Add sphinx doc for in-memory read and write - https://github.com/openmicroscopy/bioformats/pull/2812
Add Hamamatsu public sample links - https://github.com/openmicroscopy/bioformats/pull/2809
Options doc - https://github.com/openmicroscopy/bioformats/pull/2804
BDReader: Minor fix to check if Experiment.exp is a directory -
 https://github.com/openmicroscopy/bioformats/pull/2802
Imago doesn't exist anymore - https://github.com/openmicroscopy/bioformats/pull/2801
Update focalpoint link - https://github.com/openmicroscopy/bioformats/pull/2800
MIAS Reader - Unhandled Exception - https://github.com/openmicroscopy/bioformats/pull/2799
Enable forward compatibility OBF - https://github.com/openmicroscopy/bioformats/pull/2810

### CODE REVIEW

**Comment by hflynn** [comment]
Don't see any issues from a quick read through and the formatting looks like it should be fine but I'll check it properly tomorrow

File: docs/sphinx/about/whats-new.rst [hunk 1]
```diff
@@ -1,6 +1,34 @@
...
+
+* updated external homepage link for FocalPoint
+* removed imago from list of visualization and analysis applications as it is no
+  longer available from the Mayachitra website
+* added links to public sample files for Hamamatsu ndpi and Hamamatsu VMS
+* listed OpenSlide as available software for supported formats
+* added a new developer page detailing in-memory reading and writing
+* updated the Bio-Formats API versioning policy which now follows strict 
+  semantic versioning
+* a new options page has been added detailing the usage of configurable format specific
```
Comment (mtbc) [inline docs/sphinx/about/whats-new.rst]: hyphenate format-specific

File: docs/sphinx/about/whats-new.rst [hunk 2]
```diff
@@ -1,6 +1,34 @@
...
+* BD Pathway
+   - added fix to check if ``Experiment.exp`` is a directory or an experiment file
+* Imspector OBF
+   - enabled forward compatibility for future versions as the OBF format is backwards 
+     compatible (thanks to Bjoern Thiel)
+
+Documentation improvements:
+
+* updated external homepage link for FocalPoint
+* removed imago from list of visualization and analysis applications as it is no
```
Comment (rleigh-codelibre) [inline docs/sphinx/about/whats-new.rst]: Does `imago` require capitalisation?

File: docs/sphinx/about/whats-new.rst [hunk 3]
```diff
@@ -1,6 +1,34 @@
...
+  longer available from the Mayachitra website
+* added links to public sample files for Hamamatsu ndpi and Hamamatsu VMS
```
Comment (rleigh-codelibre) [inline docs/sphinx/about/whats-new.rst]: Should `ndpi` be uppercase to match `VMS`?

File: docs/sphinx/about/whats-new.rst [hunk 4]
```diff
@@ -1,6 +1,34 @@
...
+* listed OpenSlide as available software for supported formats
+* added a new developer page detailing in-memory reading and writing
+* updated the Bio-Formats API versioning policy which now follows strict
```
Comment (rleigh-codelibre) [inline docs/sphinx/about/whats-new.rst]: Comma after `policy`?

File: docs/sphinx/about/whats-new.rst [hunk 5]
```diff
@@ -1,6 +1,34 @@
...
+
+* updated external homepage link for FocalPoint
+* removed imago from list of visualization and analysis applications as it is no
+  longer available from the Mayachitra website
+* added links to public sample files for Hamamatsu ndpi and Hamamatsu VMS
+* listed OpenSlide as available software for supported formats
+* added a new developer page detailing in-memory reading and writing
+* updated the Bio-Formats API versioning policy which now follows strict 
+  semantic versioning
+* a new options page has been added detailing the usage of configurable format-specific
```
Comment (rleigh-codelibre) [inline docs/sphinx/about/whats-new.rst]: Comma after `added`?

File: docs/sphinx/about/whats-new.rst [hunk 6]
```diff
@@ -1,6 +1,34 @@
...
+
+File format improvements:
+
+* MIAS (Maia Scientific)
+   - added a fix for a possible exception when image files are not found under 
+     channel-specific subdirectories
+* BD Pathway
+   - added fix to check if ``Experiment.exp`` is a directory or an experiment file
+* Imspector OBF
+   - enabled forward compatibility for future versions as the OBF format is backwards
```
Comment (rleigh-codelibre) [inline docs/sphinx/about/whats-new.rst]: Comma after `versions`?

**Comment by dgault** [comment]
For both the ndpi and imago capitalisation I was basing it on the existing text from the formats pages. Im more than happy to update them just wanted to make sure that the capitalised version is correct? For ndpi a quick google also shows it as NDPI or NDPi, imago I can find with a mix of Imago or Mayachitra imago

**Comment by rleigh-codelibre** [comment]
All looks fine to me with these last changes, thanks.

**Comment by hflynn** [comment]
Yep, good to go I think

**Comment by hflynn** [comment]
http://www.openmicroscopy.org/site/support/bio-formats5.4-staging/about/whats-new.html is updated and looks good, ready to merge.

**Pull request merged**